### PR TITLE
Update docs and add release notes

### PR DESCRIPTION
**Pull request by mudler**
- Adds section explaining how to build KubeCF from source ( we assume the reader is familiar with bazel, we shouldn't!)
- Add release notes
- Update docs and sync them with KubeCF content.

### CODE REVIEW

File: content/en/blog/releases/kubecf-1.0.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+---
+title: "KubeCF Release v1.0"
+linkTitle: "KubeCF Release v1.0"
+date: 2020-03-19
+description: >
+  KubeCF 1.0 has been released!
+---
+KubeCF first major release is out 🥳 ...
+
+<img src="https://media.giphy.com/media/Sk5uipPXyBjfW/giphy.gif">
```
Comment (mudler) [inline content/en/blog/releases/kubecf-1.0.md]: This needs to be converted in markdown

File: content/en/blog/releases/kubecf-1.0.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+# Features
+
+- HA database configuration for CCDB and UAA based on the [upstream PXC chart](https://github.com/helm/charts/tree/master/stable/percona-xtradb-cluster)
+ [#312]
```
Comment (mudler) [inline content/en/blog/releases/kubecf-1.0.md]: Links to PR missing

**Comment by mudler** [comment]
We should also add a blog post about KubeCF that now is an official incubating project :tada: !

File: content/en/docs/Tasks/secrets.md [hunk 3]
```diff
@@ -20,14 +20,6 @@ Beyond this, the keys used to encrypt the Cloud Controller Database
 secrets of the KubeCF deployment. This means that the general process
 explained above __does not apply__ to them.
 
-
-## Secret Rotation
-
-Note, this document explains the general rotation of secrets.
-
-The instructions to rotate the CCDB keys specifically are in
-[a separate document](secret_rotation.md).
```
Comment (jimmykarily) [inline content/en/docs/Tasks/secrets.md]: Did it move here? https://github.com/cloudfoundry-incubator/kubecf-docs/blob/master/content/en/docs/Tasks/secrets.md

File: content/en/blog/releases/kubecf-1.0.1.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+description: >
+  KubeCF 1.0.1 has been released!
+---
+
+# Bug fixes
+
+- Fixes the permissions issue in autoscaler db when the pod restarts. [#467](https://github.com/cloudfoundry-incubator/kubecf/pull/467)
+- Moved pxc image to cfcontainerization organization, like all the other images. [#479](https://github.com/cloudfoundry-incubator/kubecf/pull/479)
+- The database-seeder job pod was not being deleted after KubeCF helm installation was removed with success. [#493](https://github.com/cloudfoundry-incubator/kubecf/pull/493)
+- Security group setups. [#485](https://github.com/cloudfoundry-incubator/kubecf/pull/485)
```
Comment (jimmykarily) [inline content/en/blog/releases/kubecf-1.0.1.md]: Are these missing? https://github.com/cloudfoundry-incubator/kubecf/pull/485 @fargozhu

File: content/en/docs/Tasks/secrets.md (hunk 3 above)
Comment (mudler) [reply content/en/docs/Tasks/secrets.md]: I've moved it here under one page split into paragraphs :paintbrush:

File: content/en/docs/Contribution guidelines/_index.md [hunk 5]
```diff
@@ -46,6 +46,22 @@ combinations of choices in detail, from the bottom up.
...
+
+To build KubeCF from source, ``bazel`` is required.
+You can check [the official bazel docs on how to install](https://docs.bazel.build/versions/master/install.html) it in your system.
+
+To build [KubeCF](https://github.com/cloudfoundry-incubator/kubecf) from source, you can just:
+
+```sh
+$> git clone https://github.com/cloudfoundry-incubator/kubecf
+$> cd kubecf
+$> bazel build //deploy/helm/kubecf
```
Comment (mudler) [inline content/en/docs/Contribution guidelines/_index.md]: this probably needs to be expanded a bit more. I don't think that the bazel output is easy to digest for newcomers

File: content/en/docs/Tasks/secrets.md (hunk 3 above)
Comment (mudler) [reply content/en/docs/Tasks/secrets.md]: this is a leftover of https://github.com/cloudfoundry-incubator/kubecf-docs/commit/0e72670c276dec255f2ffc0bfa6ef75c1e1b68cd

File: content/en/blog/releases/kubecf-1.0.1.md (hunk 4 above)
Comment (fargozhu) [reply content/en/blog/releases/kubecf-1.0.1.md]: @jimmykarily missing from where? The #485 was merged and shipped on 1.0.1

**Comment by mudler** [comment]
Ok to merge? I don't see any reasons to delay this

**Comment by mudler** [comment]
I would push it a bit further, and go ahead and deleting the docs from kubecf that doesn't belong there (target audience: users, operators)

File: content/en/blog/releases/kubecf-1.0.1.md (hunk 4 above)
Comment (jimmykarily) [reply content/en/blog/releases/kubecf-1.0.1.md]: @fargozhu #485 fixes: #475 and #480 (according to the PRs description). If those issue are fixed, should they be listed here, in the bug fixes of the 1.0.1 release?

**Pull request merged**